backend/main.py

```
@@ -1,21 +1,10 @@
+#importing fastapi
 from fastapi import FastAPI
-from fastapi.middleware.cors import CORSMiddleware
-from database.session import engine, Base
-from api.v1.endpoints import auth
+from typing import Optional
```

Ruff (0.8.2)

`typing.Optional` imported but unused

Remove unused import: `typing.Optional`

(F401)